dotnet/samples/KernelSyntaxExamples/Getting_Started/Step7_Observability.cs

```diff
@@ -15,10 +15,35 @@ namespace GettingStarted;
 
 public class Step7_Observability : BaseTest
 {
+    /// <summary>
+    /// Shows how to observe the execution of a <see cref="KernelPlugin"/> instance with filters.
+    /// </summary>
+    [Fact]
+    public async Task ObservabilityWithFiltersAsync()
```

Comment:
Would be interesting to have also one example of using filters without DI approach.